content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,96 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use of special
+processors, and minimize data transfers. This involves provisioning and managing
```

Comment:
rotate the use of special processors -> utilize available GPUs or share them ... or something like that